internal/providers/pke/pkeworkflow/ec2.go

```diff
@@ -17,10 +17,16 @@ package pkeworkflow
 import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/client"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
+
+	internalAmazon "github.com/banzaicloud/pipeline/internal/providers/amazon"
 )
 
+// ErrReasonStackFailed cadence custom error reason that denotes a stack operation that resulted a stack failure
+const ErrReasonStackFailed = "CLOUDFORMATION_STACK_FAILED"
```

Comment:
**Recommended**: I think this **SHOULD** be in the `cloudformation.go` file instead of the `ec2.go`, because this is not part of EC2, but it is part of CloudFormation.

Reply:
Good point. Fixed.